python/openvino_tokenizers/utils.py

```diff
@@ -290,3 +296,64 @@ def quote_meta(unquoted: Union[str, bytes]) -> str:
             symbols.append("\\")
         symbols.append(char)
     return "".join(symbols)
+
+
+def to_bytes(number: int) -> bytes:
+    return number.to_bytes(4, "little")
+
+
+class UnpackedOutputs:
+    _outputs = None
+    def __init__(self, outputs):
+        self._outputs = outputs
+
+    def outputs(self) -> List:
+        if self._outputs:
+            return self._outputs
+        else:
+            return []
```

Comment:
```suggestion
@dataclass
class UnpackedOutputs:
    outputs: List[Output] = field(default_factory=list)
```

Reply:
And add `from openvino.runtime import Output`

Reply:
That's great! will apply it

Reply:
It turned out `TypeError: 'list' object is not callable`. But i managed to do it without this w/a. I removed `.output()` everywhere where `create_string_constant_node` is used.